corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -103,12 +105,20 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None):
             if getattr(f, 'schedule', False) and f.schedule.enabled
         ])
 
-
+    vellum_base = 'corehq/apps/app_manager/static/app_manager/js/'
+    vellum_dir = 'vellum'
+    if toggles.VELLUM_BETA.enabled(domain) and isdir(join(vellum_base, 'vellum_beta')):
+        vellum_dir = 'vellum_beta'
     context = get_apps_base_context(request, domain, app)
     context.update(locals())
     context.update({
-        'vellum_debug': settings.VELLUM_DEBUG,
+        'vellum_debug': settings.VELLUM_DEBUG and not toggles.VELLUM_BETA,
         'nav_form': form,
+        'vellum_style_path': 'app_manager/js/{}/style.css'.format(vellum_dir),
+        'vellum_ckeditor_path': 'app_manager/js/{}/lib/ckeditor/'.format(vellum_dir),
+        'vellum_js_path': 'app_manager/js/{}/src'.format(vellum_dir),
+        'vellum_main_components_path': 'app_manager/js/{}/src/main-components.js'.format(vellum_dir),
+        'vellum_local_deps_path': 'app_manager/js/{}/src/local-deps.js'.format(vellum_dir),
```

Comment:
yeah, if there's a more succinct way to do this, i'm interested. spent a bunch of time fighting with django template language.